src/formatDate.js

```diff
@@ -8,7 +8,30 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
```

Comment:
Using `pop()` to extract the separator from `fromFormat` modifies the original array by removing its last element. This could lead to unexpected behavior if the separator is not intended to be the last element. Consider using a different approach to extract the separator without modifying the array.